projects/1.16/assets/litematica/litematica/lang/zh_cn.json

```
@@ -705,7 +705,7 @@
   "pasteReplaceBehavior": "粘贴替换模式",
   "The behavior of replacing existing blocks\nin the Paste schematic tool mode": "在粘贴原理图工具模式中替换现有方块的方式",
   "pickBlockEnabled": "方块抓取启用",
-  "Enables the schematic world pick block hotkeys.\nThere is also a hotkey for toggling this option to toggle those hotkeys... o.o": "启用投影原理图方块选择快捷键。\n还有一个热键来切换这个选项来切换这些热键的工作模式\n... o.o",
+  "Enables the schematic world pick block hotkeys.\nThere is also a hotkey for toggling this option to toggle those hotkeys……o.o": "启用投影原理图方块选择快捷键。\n还有一个热键来切换这个选项来切换这些热键的工作模式\n……o.o",
```

~以英文原文作key，也是服了~
key里面需要还原。
```suggestion
  "Enables the schematic world pick block hotkeys.\nThere is also a hotkey for toggling this option to toggle those hotkeys... o.o": "启用投影原理图方块选择快捷键。\n还有一个热键来切换这个选项来切换这些热键的工作模式\n……o.o",
```

---

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

~以英文原文作key，也是服了~
key里面需要还原。
```suggestion
  "Enables the schematic world pick block hotkeys.\nThere is also a hotkey for toggling this option to toggle those hotkeys... o.o": "启用投影原理图方块选择快捷键。\n还有一个热键来切换这个选项来切换这些热键的工作模式\n……o.o",
```